### PR TITLE
Implement table of data sets on interactive data page

### PR DESCRIPTION
**Pull request by nholden**
This PR implements the table of data sets on the new interactive data page. It's based on [these designs in Figma](https://www.figma.com/file/gDZgUV7yw7PwD3hQQC8V7o/TJI?node-id=2%3A518).

### 🚧 Todo in future PRs
- [ ] Implement the hero section (@judan will handle this)
- [ ] Add query params to links to the existing explore the data page so the user lands on the desired dataset (@LayaTaal will handle this)
- [ ] Double check copy and last updated dates (@EvaRuthM?)

### CODE REVIEW

File: components/DataTable.js [hunk 1]
```diff
@@ -1,36 +1,127 @@
...
+        display: flex;
+      }
+
+      .dataset-title {
+        color: ${props => props.theme.colors.black};
+        font-weight: ${props => props.theme.fontWeights.bold};
+        letter-spacing: ${props => props.theme.letterSpacings.medium};
+
+        @media (min-width: ${props => props.theme.medium}) {
+          font-size: 22px;
```
Comment (LayaTaal) [inline components/DataTable.js:80]: I haven't looked into the Figma files much, so I just want to make sure these font sizes align with our existing sizes well. Should we go set these up as variables?

I know we have talked about Julia doing a full audit of the size to help establish a more uniform style guide, so maybe font sizes can be included in that if we are starting to build up a lot of variation.

**Review by LayaTaal — APPROVED** [review]
Looks good @nholden! 👍 I just left one question below.

File: components/DataTable.js (hunk 1 above)
Comment (nholden) [reply components/DataTable.js]: I like that idea! Not sure if these are the sizes we'll want to stick with, but in https://github.com/texas-justice-initiative/website-nextjs/pull/471/commits/f6544c4abc692e38a543ec5d51fe4e197905864a, I added variables for the different font sizes and line heights I saw in Figma, using the values we use for our normal body text as the `md` size for both. What do you think?

Comment (LayaTaal) [reply components/DataTable.js]: Yeah I think this is great @nholden! We have them available and can address the sizes more easily this way. I like the refactor on the variables!

**Pull request merged**